airflow/sample.env

```diff
@@ -33,7 +33,10 @@ OPENEO_PYTHON_UDF_URL=http://openeo-python-udf:5000/udf  # do not change if runn
 OPENEO_R_UDF_URL=http://openeo-r-udf:5555/udf  # do not change if running the containerized one
 
 # URL of CSW server
-CSW_SERVER=http://pycsw:8000  # do not change if running the containerized one
+#CSW_SERVER=http://pycsw:8000  # do not change if running the containerized one
```

Comment:
Remove line instead of commenting it if not needed any more - is kept in history anyway

Reply:
makes sense